### PR TITLE
aneglica.json

### PR DESCRIPTION
**Pull request by vfcsz03**
(Missing Memory Imprint)

### CODE REVIEW

File: hero/angelica.json [hunk 1]
```diff
@@ -0,0 +1,544 @@
...
+      "command": "25",
+      "charm": "62",
+      "politics": "90"
+    }
+  },
+  "memoryImprint": [
+    {
+      "rank": "d",
+      "status": {
+        "type": "afr",
```
Comment (kmalone86) [inline hero/angelica.json]:
```suggestion
        "type": "efr",
```

File: hero/angelica.json [hunk 2]
```diff
@@ -0,0 +1,544 @@
...
+        "increase": 0
+      }
+    },
+    {
+      "rank": "c",
+      "status": {
+        "type": "afr",
```
Comment (kmalone86) [inline hero/angelica.json]:
```suggestion
        "type": "efr",
```

File: hero/angelica.json [hunk 3]
```diff
@@ -0,0 +1,544 @@
...
+        "increase": "3.6%"
+      }
+    },
+    {
+      "rank": "b",
+      "status": {
+        "type": "afr",
```
Comment (kmalone86) [inline hero/angelica.json]:
```suggestion
        "type": "efr",
```

File: hero/angelica.json [hunk 4]
```diff
@@ -0,0 +1,544 @@
...
+        "increase": 0
+      }
+    },
+    {
+      "rank": "a",
+      "status": {
+        "type": "afr",
```
Comment (kmalone86) [inline hero/angelica.json]:
```suggestion
        "type": "efr",
```

File: hero/angelica.json [hunk 5]
```diff
@@ -0,0 +1,544 @@
...
+        "increase": 0
+      }
+    },
+    {
+      "rank": "s",
+      "status": {
+        "type": "afr",
```
Comment (kmalone86) [inline hero/angelica.json]:
```suggestion
        "type": "efr",
```

File: hero/angelica.json [hunk 6]
```diff
@@ -0,0 +1,544 @@
...
+        "increase": 0
+      }
+    },
+    {
+      "rank": "ss",
+      "status": {
+        "type": "afr",
```
Comment (kmalone86) [inline hero/angelica.json]:
```suggestion
        "type": "efr",
```

File: hero/angelica.json [hunk 7]
```diff
@@ -0,0 +1,544 @@
...
+        "increase": 0
+      }
+    },
+    {
+      "rank": "sss",
+      "status": {
+        "type": "afr",
```
Comment (kmalone86) [inline hero/angelica.json]:
```suggestion
        "type": "efr",
```

File: hero/angelica.json [hunk 8]
```diff
@@ -0,0 +1,544 @@
...
+        }
+      ],
+      "resources": [
+        {
+          "item": "frost-rune",
+          "qty": "12"
+        },
+        {
+          "item": "greater-frost-rune",
+          "qty": "8"
```
Comment (kmalone86) [inline hero/angelica.json]:
```suggestion
          "qty": "2"
```

**Comment by vfcsz03** [comment]
Updated with kmalone86's corrections

**Pull request merged**